src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,51 @@
+.calendar {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: $calendar-gap;
+
+  width: $calendar-width;
+  padding: $calendar-paddings;
+  padding-top: 40px;
```

Comment:
you have paddings on in variables = 10px, but you want to make top 40px, so may be type
padding: 40px 10px 10px; here it will be like 40px top, 10px left-right(sides) 10px bottom

Reply:
I more tell that here isn't necessary to use any variable in this case. just use one line, and define padding once. now u have 2 lines, and one line change properties of top one. I just tell that if you have some style value only once it's better to type it, than create variable.

Reply:
ok. but in that case, I'll need to type 10px into the count of calendar width manually too, right?

Reply:
no